spec/models/user_spec.rb

```diff
@@ -39,23 +47,10 @@
     it{ is_expected.to validate_uniqueness_of(:email) }
   end
 
-  describe '.staff_array' do
-    let(:attributes) do
-      [
-        User.human_attribute_name('staff/team'),
-        User.human_attribute_name('staff/financial_board'),
-        User.human_attribute_name('staff/technical_board'),
-        User.human_attribute_name('staff/advice_board'),
-      ]
-    end
-
-    let(:expected_array) do
-      attributes.each_with_index.map { |name, index| [name, index] }
+  describe '.staff_descriptions' do
+    it 'must returns an array matching all the STAFF\'s constant keys' do
```

Comment:
Prefira usar aspas duplas aqui pra evitar o `\'`, e talvez seja melhor usar `it 'should ...` ou só `it 'returns`. `must returns` está com ortograficamente errado 😛